.github/workflows/github-action.yml

```diff
@@ -88,8 +88,15 @@ jobs:
       - name: ctest performance tests on cruxbc
         working-directory: ${{github.workspace}}/Release-build
         if: runner.os == 'Linux'
-        run: 
+        run: |
           ctest -R diff-perf-cruxbc -VV
+          git pull
+          cp -r ${{github.workspace}}/Release-build/Testing/Temporary/LastTest.log $GITHUB_WORKSPACE/Test-Suite/diff_tests/perf_history/perf-$(date +'%Y-%m-%dT%H:%M:%S').txt
+          cp -r ${{github.workspace}}/Release-build/Testing/Temporary/LastTest.log $GITHUB_WORKSPACE/Test-Suite/diff_tests/perf-latest.txt
+          cd $GITHUB_WORKSPACE/Test-Suite/diff_tests
+          python3 -m pip install --upgrade pip setuptools wheel
+          pip3 install -r ./requirements.txt
```

Comment:
can you wrap these in your python script?

Reply:
Will these installations cause the CI to slow much

Reply:
If we really wanted to, we could wrap it in the python script, but this is not recommended, as described [here](https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program). After adding `time` commands, I've found the installation time to only be a few seconds in total, so the runtime is negligible and shouldn't affect the CI runtime much.